Automattic/simplenote-android
# Add script to automate builds

This PR adds a lane to automate the pre-build checks and the build itself.
This PR builds on top of https://github.com/Automattic/simplenote-android/pull/749 and should be merged after that one. 

# To Test
1. Run `bundle install`.
2. Run `bundle exec fastlane build_release`.
3. When prompted, verify that the version is correct and confirm.
4. If prompted, update the secret file.
5. Verify that the script finishes without errors.
6. Verify that the apk has been generated in the `build` folder.
7. Test the apk.
